aksel.nav.no/website/app/dev/produktbloggen/_ui/Produktbloggen.module.css

```diff
@@ -1,9 +1,132 @@
+.overviewTitle {
+  font-size: 48px;
+  color: var(--ax-text-brand-blue);
+  text-align: center;
+}
+
+.overviewSubtitle {
+  & p {
+    color: var(--ax-text-brand-blue-subtle);
+    text-align: center;
+    max-width: 670px;
+    font-size: var(--ax-font-size-heading-small);
+  }
+}
```

Comment:
Not that big a fan of overrides like this. Does "Intro" even need to be fetched from sanity here? We could aswell just code inn the text `Skriverier fra produktutviklingsmiljøet i NAV. Har du eller teamet ditt noe å dele? Ta kontakt med Team Aksel!` into a simple `BodyLong` 🤔

Reply:
Yeah, since we already hardcode "Produktbloggen" I was suprised we had the intro fetched from Sanity.

I'll convert more to just simple BodyLongs.